### DISCUSSION TITLE
Invoke expo app from background

### DISCUSSION THREAD

**Discussion by MarouaniALA** [react-navigation/react-navigation#10289] [General]
Please provide the following:
1. SDK Version: 43
2. Platforms(Android/iOS):
3. Deeplinking, Notification Background, Invoke.
the goal is to open a page in the app when I get a notification.
in expo it works with Deeplinking in the foreground but not in the background.

```

const NOTIFICATION_TASK_NAME = 'background-notification-task';

TaskManager.defineTask(NOTIFICATION_TASK_NAME, async ({data, error}) => {

        if (error) {
            // Error occurred - check `error.message` for more details.
            return;
        }
        if (data) {
            Toast('got notif on background')// it Works
          
        const redirectUrl = Linking.createURL('FullScreenNotif', {
                queryParams: {id},
            });

          Linking.openURL(redirectUrl);
        }
    }
);

Notifications.registerTaskAsync(NOTIFICATION_TASK_NAME);

```

I found react-native-invoke (https://www.npmjs.com/package/react-native-invoke-app) which does what I want but I have to eject my app(works only in Android)

is there a way to do this in expo without ejecting my application

**Comment by MihirHundiwala** [comment]
Hi have you found any solution? I cant even console.log using TaskManager when i recieve notification in background
